test/functional/services/dashboard/visualizations.ts

```diff
@@ -109,6 +114,8 @@ export class DashboardVisualizationsService extends FtrService {
     await this.testSubjects.click('savedObjectTitlelogstash-*');
     await this.testSubjects.exists('visualizesaveAndReturnButton');
     await this.testSubjects.click('visualizesaveAndReturnButton');
+    await this.header.waitUntilLoadingHasFinished();
+    await this.dashboard.waitForRenderComplete();
```

Comment:
Same for `createAndEmbedMetric`, lets remove call to waitForRenderComplete after call to `createAndEmbedMetric` https://github.com/elastic/kibana/blob/main/test/functional/apps/dashboard/group1/create_and_add_embeddables.ts#L208

Reply:
Done in f646bb95848c25441f10c212678ecc062c771854